jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantAdapter/configSchema.ts

```diff
@@ -0,0 +1,24 @@
+import { types } from 'mobx-state-tree'
+import { ConfigurationSchema } from '@jbrowse/core/configuration'
+
+export default ConfigurationSchema(
+  'ExtendedVariantAdapter',
+  {
+    vcfGzLocation: {
```

Comment:
can this load the schema of the original VariantAdapter, instead of repeating these properties?

Reply:
Updated. We can't fully extend it as we need to change the name property to ExtendedVariantAdapter, but we can import unchanged properties piecemeal.